src/main/java/Storage.java

```diff
@@ -0,0 +1,120 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Handles all issues related to File.txt.
+ */
+public class Storage {
+    public static String filePath;
+    public static File f;
```

Comment:
Perhaps name the File variable as file. 'f' should be used for scratch variables (ie. short term use).